### ISSUE TITLE
ModuleNotFoundError: No module named 'win32api'

### ISSUE THREAD

**Issue by EnzoDeg40** [OpenNaja/cobra-tools#384]
**Describe the bug**
I installed python without additional libraries. When he suggested that I install the libraries before running the python file, he forgot to install pywin32. To solve this, a simple `pip install pypiwin32` works (not identical to `pywin32`). (https://stackoverflow.com/a/35948588/13845688)

**To Reproduce**
Command:
```bash
python .\ovl_tool_gui.py
```
Output :
```
DEBUG:root:Checking for automatic module updates
ERROR:root:imageio not found.
ERROR:root:numpy not found.
ERROR:root:PyQt5 not found.
ERROR:root:PyQt5-Frameless-Window not found.
ERROR:root:vdf not found.
ERROR:root:pillow not found.
Install the missing dependencies? (y/N)
[Type y and hit Enter]
y
...
ModuleNotFoundError: No module named 'win32api'
```

**Version Info:**
 - Python Version:  3.11.4
 - Cobra Tools Version May 10, 2024

**Comment by HENDRIX-ZT2** [comment]
Why don't you just re-install python with libraries?

**Comment by EnzoDeg40** [comment]
> Why don't you just re-install python with libraries?

Well, if your project needs pypiwin32 then you must indicate it in the project. Yes my python environment is empty, I don't want to install basic libraries which are of no use to me. It's a bit stupid to reinstall python just to have the right library that this project requires.

**Comment by HENDRIX-ZT2** [comment]
Good point - however: the requirement for win32api comes from a third-party library (qframelesswindow).
They do have it in their requirements.txt file, so something must have gone wrong on your end during installation.
https://github.com/zhiyiYo/PyQt-Frameless-Window
